canonical-web-and-design/ubuntu.com
# Fix top website 404s

## Done

- redirects for favicons.ico, favicons.png, robots.txt, humans.txt, usn/atom.xml
- added a sitemap.xml generated from https://www.xml-sitemaps.com/

# QA

- download this branch
- run the site and make sure everything looks ok
- go to:
    - http://0.0.0.0:8001/favicon.ico
    - http://0.0.0.0:8001/favicon.png
    - http://0.0.0.0:8001/humans.txt
    - http://0.0.0.0:8001/robots.txt
    - http://0.0.0.0:8001/static/files/sitemap.xml
    - http://0.0.0.0:8001/usn/atom.xml
- see that they all make sense

## Screenshots

```
               .-.
         .-'``(   )
      ,`\ \    `-`.
     /   \ '``-.   `
   .-.  ,       `___:
  (   ) :        ___
   `-`  `       ,   :
     \   / ,..-`   ,
      `./ /    .-.`
         `-..-(   )
               `-`
```
### greylog 

![image](https://user-images.githubusercontent.com/441217/59697189-54567000-91e5-11e9-9e09-c7cd1698587e.png)
